test/testclass.cpp

```diff
@@ -430,6 +430,11 @@ class TestClass : public TestFixture {
                                   "    B(const B&) {}\n"
                                   "};");
         ASSERT_EQUALS("", errout.str());
+
+        checkExplicitConstructors("struct A{"
+                                  "A(int, int y=2) {y;}"
```

Comment:
the `y;` seems redundant.

Reply:
Please indent the code. put some spaces before the constructor. It is not technically needed, but I would prefer that..